src/platform/unix/mod.rs

```diff
@@ -617,7 +613,8 @@ impl OsIpcOneShotServer {
 
             Ok((OsIpcOneShotServer {
                 fd: fd,
-            }, String::from_utf8(CStr::from_ptr(path.as_ptr() as
+                _temp_dir: temp_dir,
+            }, String::from_utf8(CStr::from_ptr(path_string.as_ptr() as
                                                 *const c_char).to_bytes().to_owned()).unwrap()))
```

Comment:
You didn't remove the back-and-forth conversion here -- but that was the whole point of shuffling around the other conversions/bindings above...

Reply:
So, this will be just
`path_string.to_string()`
right?

Reply:
Indeed :-)

Reply:
done